tracer/build/_build/Build.Shared.Steps.cs

```diff
@@ -222,7 +222,7 @@ partial class Build
     Target PublishNativeLoaderUnix => _ => _
         .Unlisted()
         .OnlyWhenStatic(() => IsLinux)
-        .After(CompileNativeLoader)
+        .After(CompileNativeLoader, CompileNativeLoaderLinux)
```

Comment:
nit: `CompileNativeLoader` includes `CompileNativeLoaderLinux` so this isn't necessary
```suggestion
        .After(CompileNativeLoader)
```

Reply:
I was using `Depends(CompileNativeLoaderLinux)` directly somewhere because for AWS Lambda it will _always_ be Linux. I don't think I'm doing that anymore.

Reply:
I take that back. I had to replace `PublishNativeTracer` and `PublishNativeLoader` with Lambda-specific versions (`PublishNativeTracerAwsLambda` and `PublishNativeLoaderAwsLambda`), so I'm calling `CompileTracerNativeSrcLinux` and `CompileNativeLoaderLinux` directly:

```
.DependsOn(CompileTracerNativeSrcLinux)   // CompileNativeSrc
.DependsOn(PublishNativeTracerAwsLambda)  // PublishNativeTracer

.DependsOn(CompileNativeLoaderLinux)      // BuildNativeLoader -> CompileNativeLoader
.DependsOn(PublishNativeLoaderAwsLambda); // BuildNativeLoader -> PublishNativeLoader
```